mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -708,16 +708,18 @@
     <!-- Text for the dialog to confirm clearing all downloads -->
     <string name="download_delete_all_dialog">Are you sure you want to clear your downloads?</string>
     <!-- Text for the snackbar to confirm that multiple downloads items have been deleted -->
```

Comment:
Comment should be updated to reflect that downloads will be "removed", vs. "deleted" (not the same definition for both)
CC @kglazko